### ISSUE TITLE
Manually Importing causes Type Error

### ISSUE THREAD

**Issue by Yihao-G** [nuxt-community/vuetify-module#160]
**Module version**
1.8.2

**Describe the bug**
Unable to set up manually importing due to type errors. 
```
nuxt.config.ts:108:13 - error TS2322: Type '{ theme: { themes: { light: { primary: Color; secondary: string; accent: string; }; }; options: { customProperties: true; }; }; treeShake: { components: string[]; directives: string[]; }; }' is not assignable to type 'Options'.
Types of property 'treeShake' are incompatible.
Type '{ components: string[]; directives: string[]; }' is not assignable to type 'boolean | { loaderOptions?: VuetifyLoaderOptions | undefined; } | undefined'.
Object literal may only specify known properties, and 'components' does not exist in type '{ loaderOptions?: VuetifyLoaderOptions | undefined; }'.
```

**To Reproduce**

1. Set the `treeShake` option to 
```
        {
            components: [
                'VSnackbar',
                'VBtn',
                'VIcon'
            ],
            directives: [
                'Scroll',
                'Touch'
            ]
        }
``` 
as in previous versions.
2. Run `nuxt-ts`
3. Got error (see above)

**Expected behavior**
No type error should happen 

**Additional context**
Adding `@ts-ignore` can temporarily bypass this error. Seems like `Options.treeShake` in options.d.ts is missing the declaration for these fields.

**Comment by kevinmarrec** [comment]
Released **1.8.3** with the fix :)

**Comment by Yihao-G** [comment]
Thank you so much. That was quick!
